modules/nodes-pods-disabling-features-cluster.adoc

```diff
@@ -5,33 +5,42 @@
 [id="nodes-pods-disabling-features-cluster-{context}"]
 = Disabling features for a cluster
 
-You can turn off a feature for all nodes in the cluster.
+You can turn Technical Preview features on and off for all nodes in the cluster
+by editing the *FeatureGates* Custom Resource, named *cluster*, in the
+*openshift-config* project.
+
+----
+$ oc get FeatureGate cluster -n openshift-config
+----
+
+[NOTE]
+====
+Turning on Technical Preview features cannot be undone and prevents upgrades.
+====
 
 .Procedure
 
 To turn off a feature for the entire cluster:
 
-. Create a Custom Resource (CR) for feature gates.
-+
-For example, the following code enables Dynamic Auditing and disables Custom Pod DNS:
+. Edit the Custom Resource (CR) for feature gates.
 +
 [source,yaml]
 ----
 apiVersion: config.openshift.io/v1
 kind: FeatureGate
 metadata:
-  name: cluster-feature-gates
+  name: cluster
+  namespace: openshift-config
 spec:
-  enabled:           <1>
-  - DynamicAuditing
-  disabled:          <2>
-  - CustomPodDNS
+  featureSet: "TechPreviewNoUpgrade"
+     Enabled:  <1>
```

Comment:
everything on this line and later needs to be remove from the CR.  `featureSet` is a string field now that is either `Default` or `TechPreviewNoUpgrade`.  Those correspond to predetermined (by us) lists of a feature gates.

Reply:
@sjenning Do you have an updated CR that I can use?